Microsoft.jQuery.Unobtrusive.Ajax.nuspec

```diff
@@ -0,0 +1,28 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<package xmlns="http://schemas.microsoft.com/packaging/2013/05/nuspec.xsd">
+  <metadata>
+    <id>Microsoft.jQuery.Unobtrusive.Ajax</id>
+    <version>3.2.4</version>
+    <title>Microsoft jQuery Unobtrusive Ajax</title>
+    <authors>Microsoft</authors>
+    <owners>Microsoft,aspnet</owners>
+    <requireLicenseAcceptance>true</requireLicenseAcceptance>
+    <licenseUrl>http://www.microsoft.com/web/webpi/eula/net_library_eula_ENU.htm</licenseUrl>
+    <projectUrl>https://www.asp.net/</projectUrl>
+    <iconUrl>http://go.microsoft.com/fwlink/?LinkID=288859</iconUrl>
+    <description>jQuery plugin that unobtrusively sets up jQuery Ajax.</description>
+    <summary>jQuery plugin that unobtrusively sets up jQuery Ajax.</summary>
+    <releaseNotes>Please visit http://go.microsoft.com/fwlink/?LinkId=389866 to view the release notes.</releaseNotes>
+    <copyright>© Microsoft Corporation. All rights reserved.</copyright>
+    <tags>jQuery Unobtrusive Ajax</tags>
+    <dependencies>
+      <dependency id="jQuery" version="1.8.0" />
+    </dependencies>
+  </metadata>
+    <files>
+    <file src="dist\jquery.unobtrusive-ajax.js" target="Content"/>
+    <file src="dist\jquery.unobtrusive-ajax.min.js" target="Content"/>
+    <file src="LICENSE.txt" target=""/>
+    <file src="README.md" target=""/>
```

Comment:
Is this needed?

Reply:
It's not. Wasn't sure if we wanted it as part of the NuGet package